changelog.txt

```diff
@@ -47,6 +47,7 @@ Fixed
 
 - Ensure that directives are always evaluated before running or submitting (#408, #409).
 - Cache the fully qualified domain name during environment detection to fix a performance issue on macOS (#339, #394).
+- Ensure that ``next`` command line argument does not display eligible jobs for unequal operations (#443).
```

Comment:
In my understanding, this bug was never released, so this does not need to be marked as a "fix." I would just put this number in the list above with all the other aggregation-related PRs, which is how I've handled some similar PRs:
```
- Added internal support for aggregates of jobs (#334, #348, #351, #364, #383, #390, #415, #422, #430, #443).
```

Reply:
I think this bug was released in `0.11.0`.

Reply:
Here is the link to the code in the branch `release/0.11.0`: https://github.com/glotzerlab/signac-flow/blob/release/0.11.0/flow/project.py#L3659